CP1404_practicals/prac_05/state_names.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
Consider rewriting this module docstring to explain what the finished code does.